### ISSUE TITLE
Problems creating new project from File / New Project

### ISSUE THREAD

**Issue by cstockloew** [universAAL/tools.eclipse-plugins#80]
_Originally Opened: @Alfiva (2012-02-02 10:56:10_)
_Originally Closed: 2012-02-03 11:54:14_

When trying to start the Wizard from the File / New Project, it is listed under AAL Studio as assumed But it does not start. When I try to run from another Eclipse instance, I get an error message for the following line:
 PlatformUI.getWorkbench().getHelpSystem()
  .setHelp(getShell(), Activator.PLUGIN_ID + &quot;.help_project&quot;);
Or rather, there is a null pointer exception inside the setHelp()
The strange thing is that when I start it from the AAL Studio menu, it does start.

Code says //TODO: I don´t know why this doesn´t work with getShell in NewItemWizad.addPages (see Project wizard)

--

From: _this issue has been automatically imported from our old issue tracker_

**Comment by cstockloew** [comment]
_Original author: @Alfiva (2012-02-03 11:54:14)_

Fixed: Moved the creation of the help from the main wizard classes to the page classes.
However it is strange because I remember I tried this and it worked, at least at first, maybe it changed in Eclipse 3.7.
There is still some faulty behavior: when clicking on the help button, at least in the mock-up Eclipse used to test the wizard, there appears an error while trying to read some offical help index. The consequence seems to be that the help panel does not show the link to the uAAL wiki page as it should. Clicking on the help button again fixes this. This seems to be a wrong behaviour by Eclipse itself, I cannot do anything else.
